tests/test_process.py

```diff
@@ -90,6 +91,207 @@ def test_upload_dataversion_metadata_without_team_images_id(self, syn: Any):
         )
 
 
+class TestApplyCustomTransformations:
+    @pytest.fixture(scope="function", autouse=True)
+    def standard_transform_function(self):
+        """mock simple transform function"""
+
+        def _standard_transform_function(
+            df: pd.DataFrame, dataset_name: str, datasets: Dict[str, pd.DataFrame]
+        ) -> pd.DataFrame:
+            """mock simple transform function"""
+            return df.assign(test_col=2)
+
+        return _standard_transform_function
+
+    @pytest.fixture(scope="function", autouse=True)
+    def special_transform_function(self):
+        """mock transform function that takes test_threshold as an argument"""
+
+        def _mock_transform_with_args(
+            df: pd.DataFrame, test_threshold: int
+        ) -> pd.DataFrame:
+            """mock transform function that takes test_threshold as an argument"""
+            return df.assign(new_key=1)
```

Comment:
Can you assign `new_key` to `test_threshold` instead of a hard-coded value, so we can test that the argument got passed in and used correctly?